commands.go

```diff
@@ -28,8 +28,9 @@ const (
 	deleteUserMessageType = messaging.MessageType(0x32)
 
 	// Shard messages
-	createShardGroupIfNotExistsMessageType = messaging.MessageType(0x40)
-	deleteShardGroupMessageType            = messaging.MessageType(0x41)
+	createShardGroupIfNotExistsMessageType    = messaging.MessageType(0x40)
```

Comment:
Also, you should note that since we have a policy regarding shard group duration, you know both the start-time and end-time of a shard group. We can have a single "create shard" message, which has database, retention policy, group ID (which is basically the group ID of the group before the one being created).  In fact the message doesn't even need the group ID, the message should just be "create this group of shards with this start-time, this end-time, for this database, and this retention policy".